### ISSUE TITLE
Port Website to something other than Ruby

### ISSUE THREAD

**Issue by cosmiccoincidence** [RE-SS3D/SS3D-Website#79]
## Summary

Ruby has been a pain in the ass benefits know one really knows how to use it and not that many people have Linux or want to run WSL just to review basic markdown/html blog posts.

## Goal

Convert the website to using something other than Ruby.

Like this: https://github.com/getzola/zola

**Comment by cosmiccoincidence** [comment]
Completing this also closes #53

**Comment by cosmiccoincidence** [comment]
High priority cause I cant test the website locally. I'm constantly fumbling around trying to get the right version of the right ruby gems for the last 4 years, to the point I just refuse to spend time I don't have figuring that out anymore. For the past 9 months or so I've just been developing straight on the live website, which is obviously terrible practice, but what else am I suppose to do if I cannot test the website locally?

**Comment by cosmiccoincidence** [comment]
$60 bounty on this!

**Comment by sirajlekhak** [comment]
hey! @cosmiccoincidence  if I help you getting the version of Ruby and launching it into your Local machine. Will I get $60 bounty ?

**Comment by cosmiccoincidence** [comment]
> hey! @cosmiccoincidence if I help you getting the version of Ruby and launching it into your Local machine. Will I get $60 bounty ?

No. The task is to convert our website to a different, easier to use platform. Also this task has already been assigned to someone and is nearly completed now, so it is no longer available, sorry.
